app/models/secret_storage.rb

```diff
@@ -50,6 +50,43 @@ def self.keys
     end
   end
 
+  class HashicorpVault
+
+    SECRET_BACKEND ='secret/'.freeze
```

Comment:
I guess that would make fetching secrets kind of ugly because we would have to use samson- all the time ...